pingsrv.go

```diff
@@ -57,8 +58,9 @@ func (s *pingSrv) Ping(c context.Context, in *fgrpc.PingMessage) (*fgrpc.PingMes
 	return &out, nil
 }
 
-func pingServer(port int) {
-	socket, err := net.Listen("tcp", fmt.Sprintf(":%d", port))
+func pingServer(port string) {
+	port = fnet.NormalizePort(port)
+	socket, err := net.Listen("tcp", port)
```

Comment:
👍 ain't it nice we made that function ?

Reply:
Good call indeed. I appreciate all your guidance.